src/app-layout/utils/use-drawers.ts

```diff
@@ -82,16 +80,14 @@ export function useDrawers(
     drawers: ownDrawers,
     __disableRuntimeDrawers: disableRuntimeDrawers,
   }: InternalDrawerProps & { __disableRuntimeDrawers?: boolean },
-  toolsProps: ToolsProps,
-  defaults: AppLayoutState
+  toolsProps: ToolsProps
 ) {
-  const isMobile = useMobile();
   const toolsDrawer = getToolsDrawerItem(toolsProps);
 
   const [activeDrawerId, setActiveDrawerId] = useControllable(
     ownDrawers?.activeDrawerId,
     ownDrawers?.onChange,
-    !isMobile && !toolsProps.toolsHide && toolsProps.toolsOpen && defaults.toolsOpen ? TOOLS_DRAWER_ID : undefined,
+    !toolsProps.toolsHide && toolsProps.toolsOpen ? TOOLS_DRAWER_ID : undefined,
```

Comment:
There was a logical bug in this code. `toolsProps.toolsOpen && defaults.toolsOpen`. It should be `toolsProps.toolsOpen || defaults.toolsOpen`.

But `defaults` is not needed here at all, because it is already applied in a different place: https://github.com/cloudscape-design/components/blob/a7a9038f5960a784fce5c047fc5c8e7919b21acb/src/app-layout/index.tsx#L142-L146

Codecov complained that I have a condition without a test coverage and I removed it entirely